mypy_primer.py

```diff
@@ -1763,6 +1775,19 @@ def inner() -> int | None:
         pip_cmd="{pip} install pydantic beartype hydra-core",
         cost=30,
     ),
+    Project(
+        location="https://github.com/Avasam/Auto-Split",
+        mypy_cmd="{mypy} src",
+        pip_cmd="{pip} install -r scripts/requirements.txt types-d3dshot types-keyboard"
```

Comment:
I believe this is what @hauntsaninja is suggesting when he says you could "hardcode" the dependencies here -- most of the dependencies you have in your `scripts/requirements.txt` file don't have inline types (so they're irrelevant to type-checking), and will only slow `mypy_primer` down. By just "hardcoding" the necessary dependencies in here, we can speed up mypy_primer and also solve the platform-compatibility issue for macOS :)

```suggestion
        pip_cmd="{pip} install -r numpy certifi packaging types-d3dshot types-keyboard"
```

Reply:
(You can probably do a similar thing for some of the other projects you're adding here as well :)